client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -175,7 +175,7 @@ export class List extends React.Component {
 
 				{ this.domainCreditsInfoNotice() }
 
-				<SectionHeader label={ headerText }>{ this.headerButtons() }</SectionHeader>
+				<SectionHeader>{ this.headerButtons() }</SectionHeader>
```

Comment:
I think line 166 can be removed then since it's only being used on line 171, and the use there can be replaced?

Reply:
I opted to keep it this way since it keeps the DocumentHead line easier to read; line 166 is quite long. Good eye, though!